crates/wast/src/lib.rs

```diff
@@ -140,6 +140,11 @@ impl Error {
         }
     }
 
+    /// Return the `Span` for this error.
+    pub fn span(&self) -> &Span {
+        &self.inner.span
```

Comment:
I chose to return a reference to `Span` here since I figured `Span` may not always be `Copy` in the future. But I can change that if you'd prefer.

Reply:
Ah yeah returning a bare `Span` without a reference here should be fine.